challenges/objects-arrays.js

```diff
@@ -98,6 +134,7 @@ The zoos need a list of all their animal's names (names only, not scientific) co
 */
 
 const lowerCase = [];
+zooAnimals.map((animal) => lowerCase.push(`${animal.animal_name.toLowerCase()}`))
```

Comment:
Because `.map()` returns a new array without mutating the original, we can refactor this to:
```const lowerCase = zooAnimals.map(animal=> animal.animal_name.toLowerCase())```